api/PointerEvent.json

```diff
@@ -723,7 +723,7 @@
               "version_added": "58"
             },
             "edge": {
-              "version_added": false
+              "version_added": "18"
```

Comment:
`new PointerEvent('type').twist` confirmed 0 in Edge 18 and undefined in Edge 17.